adapter/internal/eventhub/subscription.go

```diff
@@ -272,7 +274,22 @@ func retrieveAPIListFromChannel(c chan response) {
 					}
 					if _, ok := APIListMap[response.GatewayLabel]; !ok {
 						// During the startup
-						APIListMap[response.GatewayLabel] = newResponse.(*types.APIList)
+						// When GA is enabled need to load only the subscription data which are related to API UUIDs received
+						// from the GA.
+						if initialAPIUUIDListMap != nil {
+							newEmptyResponse := reflect.New(responseType).Interface()
+							APIListMap[response.GatewayLabel] = newEmptyResponse.(*types.APIList)
+							for i, api := range apiListResponse.List {
+								if _, ok := initialAPIUUIDListMap[api.UUID]; ok {
+									APIListMap[response.GatewayLabel].List = append(APIListMap[response.GatewayLabel].List,
+										apiListResponse.List[i])
+								}
+							}
+						} else {
+							// When GA is disabled load all the subscription data
+							APIListMap[response.GatewayLabel] = newResponse.(*types.APIList)
+						}
+
```

Comment:
Remove the empty line.

Reply:
Fixed with 24fe37a413fa1712e45dd74923e479d8e953b024